app/views/admin/creator_settings/_form.html.erb

```diff
@@ -83,9 +83,10 @@
 </div>
 
 <div class="crayons-field mt-6 align-left">
-  <fieldset aria-describedby="section-description">
+  <fieldset aria-describedby="terms-description">
     <legend class="crayons-field__label mb-2">Finally, please agree to the following:</legend>
     <div>
+    <p id="terms-subtitle" class="crayons-field__description">You will have the opportunity to establish the Code of Conduct and Terms and Conditions for the users of your Forem during the setup process.</p>
```

Comment:
In reading the text, I was assuming that immediately following "Finally, please agree to the following:" would be a bulleted list.  Would it make sense to move the "You will have the opportunity…" to after the bulleted list.  Or before?